### ISSUE TITLE
attr_wrapper option

### ISSUE THREAD

**Issue by Nivl** [jessemiller/HamlPy#103]
Hi,

Is there any plans to add the [attr_wrapper](http://rdoc.info/github/haml/haml/Haml/Options:attr_wrapper) option?
Because right now, it's not very easy to deal with translatable attributes value (like title, placeholder, …) :

``` haml
%input{'placeholder': "{% trans 'Username' %}",
       'id'         : 'id_username',
       'name'       : 'username',
       'type'       : 'text'}
```

The french for "Username" is "Nom d'utilisateur", so this code will be parsed as:

``` html
<input id='id_username' type='text' placeholder='Nom d'utilisateur' name='username' />
```

Which will break the tag.

**Comment by bcoughlan** [comment]
`trans` issues has come up a number of times but I'm afraid I'm unfamiliar with Django internationalisation and don't fully understand the problem. How is this usually handled in HTML with Django where the apostrophe would still need to be escaped? Is there no option in trans to use HTML entities rather than printing the apostrophes?

**Comment by Nivl** [comment]
Django won't escape the result of `trans` (and the filters don't work on tags), so each time you'll have an apostrophe in your strings, it will break the HTML element. Usualy, we switch between double/single quote depending of the text we want to translate, but we can't here.

With `attr_wrapper`, the users will be able to chose what they want to wrap the strings (actually, wrapping with double quotes is better since it's used the same way in all languages. If you don't have any double quotes in the string you want to translate, you probably won't have them in the translated string). In the worst case (when a string doesn't works with attr_wrapper), we can still translate that way : `{{ _("String")|force_escape }}`, which always works, but is very ugly and not DRY at all.

**Comment by bcoughlan** [comment]
Fixed by #106
